GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2713,8 +2703,8 @@ private void UpdateAuthorInfo()
             ThreadHelper.FileAndForget(async () =>
                 {
                     // Do not cache results in order to update the info on FormActivate
-                    string userName = Module.GetEffectiveGitSetting(SettingKeyString.UserName, cache: false);
-                    string userEmail = Module.GetEffectiveGitSetting(SettingKeyString.UserEmail, cache: false);
+                    string userName = Module.GetEffectiveGitSetting(SettingKeyString.UserName, cache: false) ?? "USER NOT CONFIGURED";
+                    string userEmail = Module.GetEffectiveGitSetting(SettingKeyString.UserEmail, cache: false) ?? "E-MAIL NOT CONFIGURED";
```

Comment:
I'm not a fan of the screaming strings. Maybe something like this?
```suggestion
                    string userName = Module.GetEffectiveGitSetting(SettingKeyString.UserName, cache: false) ?? "<username not configured>";
                    string userEmail = Module.GetEffectiveGitSetting(SettingKeyString.UserEmail, cache: false) ?? "<email not configured>";
```

Reply:
changed, but using "/" instead of "<>" which occurs in the e-mail address.